lib/text/sbv_text_parser.js

```diff
@@ -0,0 +1,106 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.text.SbvTextParser');
+
+goog.require('goog.asserts');
+goog.require('shaka.text.TextEngine');
+goog.require('shaka.text.VttTextParser');
+goog.require('shaka.util.BufferUtils');
+goog.require('shaka.util.StringUtils');
+
+
+/**
+ * @implements {shaka.extern.TextParser}
+ * @export
+ */
+shaka.text.SbvTextParser = class {
+  /** */
+  constructor() {
+    /**
+     * @type {!shaka.extern.TextParser}
+     * @private
+     */
+    this.parser_ = new shaka.text.VttTextParser();
+  }
+
+  /**
+   * @override
+   * @export
+   */
+  parseInit(data) {
+    goog.asserts.assert(false, 'SubViewer does not have init segments');
+  }
+
+  /**
+   * @override
+   * @export
+   */
+  parseMedia(data, time) {
+    const SbvTextParser = shaka.text.SbvTextParser;
+    const BufferUtils = shaka.util.BufferUtils;
+    const StringUtils = shaka.util.StringUtils;
+
+    // Get the input as a string.
+    const str = StringUtils.fromUTF8(data);
+
+    const vvtText = SbvTextParser.sbv2webvtt(str);
+
+    const newData = BufferUtils.toUint8(StringUtils.toUTF8(vvtText));
+
+    return this.parser_.parseMedia(newData, time);
```

Comment:
Using the WebVTT parser to parse this is risky since WebVTT has markup in it while SubViewer either doesn't or is in a different format.  It would be better to just parse it here since it's a simple format.  Also, it means this will work even if WebVTT is removed from the build.

Reply:
I agree, tomorrow I will implement it in a way that does not depend on WebVTT.

Reply:
Done!